conference/DS/ets.yml

```diff
@@ -0,0 +1,17 @@
+- title: ETS
+  description: IEEE European Test Symposium
+  sub: DS
+  rank: C
+  dblp: ets
+  confs:
+    - year: 2024
+      id: ets2024
+      link: https://ets24.ewi.tudelft.nl/
+      timeline:
+        - deadline: '2023-12-08 00:00:00'
```

Comment:
For now, we recommend using `abstract_deadline` to indicate the submission of abstracts, please refer to https://github.com/ccfddl/ccf-deadlines/commit/b7291367dfd8b4a8529976073f0d4cba658a9a59 for a fix, thx~

Reply:
Thanks for the heads up, it's been modified.